R/create_single_dose_dataset.R

```diff
@@ -348,29 +348,30 @@ dose_freq_lookup <- tibble::tribble(
 #'   )
 #' )
 #'
-#' # Example derive a single dose dataset with imputations
+#' # Example - derive a single dose dataset with imputations
 #'
 #' # For either single drug administration records, or multiple drug administration
 #' # records covering a range of dates, fill-in of missing treatment end datetime
-#' # EXENDTC by substitution with an acceptable alternate, for example date of
-#' # death, date of datacut may be required. This programming strategy allows the
-#' # maximum possible number of single dose records to be derived. The example R
-#' # script requires the date of datacut `DCUTDT` to be specified correctly, or
+#' # `EXENDTC` by substitution with an acceptable alternate, for example date of
+#' # death, date of datacut may be required. This example shows the
+#' # maximum possible number of single dose records to be derived. The example
+#' # requires the date of datacut `DCUTDT` to be specified correctly, or
 #' # if not appropriate to use `DCUTDT` as missing treatment end data and missing
 #' # treatment end datetime could set equal to treatment start date and treatment
 #' # start datetime. ADSL variables `DTHDT` and `DCUTDT` are preferred for
 #' # imputation use.
 #' #
 #' # All available trial treatments are included, allowing multiple different
-#' # last dose variables to be created in ad_adae.R if required.
+#' # last dose variables to be created in for example `use_ad_template("ADAE")`
+#' # if required.
 #'
 #' adsl <- tribble(
 #'   ~STUDYID, ~USUBJID, ~DTHDT,
-#'   "01", "01-701-1211", ymd("2013-01-14"),
-#'   "01", "01-710-1083", ymd("2013-08-02"),
-#'   "01", "01-704-1445", ymd("2014-11-01"),
-#'   "01", "01-701-1015", NA,
-#'   "01", "01-701-1023", NA
+#'   "01", "1211", ymd("2013-01-14"),
+#'   "01", "1083", ymd("2013-08-02"),
+#'   "01", "1445", ymd("2014-11-01"),
+#'   "01", "1015", NA,
+#'   "01", "1023", NA
 #' )
 #'
 #' ex <- tribble(
```

Comment:
Should one of these lines have NA for EXENDTC to be replaced by DTHDT or DCUTDT?

Reply:
I put in two with NA.